coralproject/talk
# Connect reset to fe

## What does this PR do?

You can now fully update your password from the UI.

## How do I test this PR?

- be logged out
- try to login and click "Forgot your password"
- enter your email
- click the link in the email
- enter your new password and confirmation
- you should be redirected, where you'll have to log in.

### Also

I'll probably have to make it so you're automagically logged in, but I wanted to get this out first.

@coralproject/tech
